api/HTMLImageElement.json

```diff
@@ -669,6 +669,56 @@
           }
         }
       },
+      "loading": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "76"
```

Comment:
Actually this landed in 77: https://storage.googleapis.com/chromium-find-releases-static/0fc.html#0fc50ca2d3c81f74f73aae8aab6079a516d9b8fb